packages/mui-material/package.json

```diff
@@ -61,9 +61,9 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.21.0",
-    "@mui/base": "5.0.0-alpha.122",
+    "@mui/base": "5.0.0-alpha.123",
     "@mui/core-downloads-tracker": "^5.11.14",
```

Comment:
Shouldn't this change as well? cc @michaldudak

Reply:
Fixed!